src/consumer.ts

```diff
@@ -0,0 +1,11 @@
+import { Message } from "./producer"
+
+export type ConsumerFunc = (message: Message) => void
+
+export class Consumer {
+  constructor(readonly handle: ConsumerFunc) {}
+
+  async close(): Promise<void> {
+    throw new Error("Method not implemented.")
+  }
```

Comment:
I think it was put there at the beginning, but I don't know if it makes sense. There is no "close" in the producer. I asked Alberto and according to him at the moment it could even be removed

Reply:
@albertobarrila @ggllcu I suggest verifying the list of commands and seeing if there is a command to "stop consuming". If there, I suggest adding a card. if There isn't, I suggest removing this command in the next PR. In the meanwhile, I merge this PR.

Reply:
@albertobarrila @ggllcu I have created this Issue https://github.com/coders51/rabbitmq-stream-js-client/issues/77